assets/css/media.css

```diff
@@ -0,0 +1,343 @@
+    font-family: 'Maven Pro', sans-serif;
+  }
+
+  body header .flex nav .nav-li-end li a:hover {
+    color: #fff;
+  }
+
+  section.last-news ul {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  main section.fundings ul {
+    flex-direction: row;
+  }
+
+  body header .flex nav > ul:last-child {
+    height: calc(100% - 32px);
+    flex-direction: row;
+    width: 100%;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  main section.fundings ul li {
+    padding: 10px 8px 0 0;
+  }
+
+  body header .flex nav > ul:last-child li {
+    justify-content: center;
+    padding: 0 1em;
+    font-size: 15px;
+    font-family: 'Ropa Sans', sans-serif;
+  }
+
+  body header .flex nav > ul:last-child li:last-child {
+    justify-content: flex-start;
+    flex: 1;
+    width: 100%;
+    transform: rotateY(180deg);
+  }
+
+  body header .flex nav > ul:last-child a:hover {
+    color: #000;
+  }
+
+  body header .flex nav > ul:last-child li:last-child a {
+    width: 30px;
+    height: 48px;
+    border-right: 1px solid #e2e2e2;
+  }
+
+  body header div.register {
+    height: 60px;
+    justify-content: flex-start;
+  }
+
+  body header div.register .logo-bottom {
+    width: 95px;
+    z-index: 1;
+    display: block;
+  }
+
+  /* main */
+  main {
+    top: 140px;
+  }
+
+  main section.hero-section {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    max-height: 600px;
+    grid-gap: 8px;
+  }
+
+  .hero-section .hero-grid {
+    grid-column: span 2;
+    grid-row: span 2;
+    max-height: 100%;
+  }
+
+  .hero-section > div:nth-child(2) {
+    margin-bottom: 0;
+  }
+
+  .hero-section > div:nth-child(2) > article,
+  .hero-section > div:nth-child(3) > article {
+    font-size: 1.2em;
+  }
+
+  .w1024-pic {
+    display: none;
+  }
+
+  main section.hero-section div .a-link {
+    justify-content: flex-end;
+    min-height: 100%;
+  }
+
+  main section.fundings div.fund-blocks > div {
+    flex-basis: 33.3%;
+  }
+
+  main section.fundings .fund-blocks > div:last-child {
+    display: block;
+  }
+
+  .hero-grid > article:nth-child(1) > h2:nth-child(2) > a:nth-child(1) {
+    font-size: 2em;
+  }
+
+  .hero-section > div > article:nth-child(1) > h2 {
+    font-size: 1.2em;
+  }
+
+  section.last-news ul li.flex {
+    flex-direction: column;
+    flex-basis: calc(33.3% - 8px);
+    margin: 4px;
+  }
+
+  section.last-news ul li.flex .news-img {
+    padding-top: 56.25%;
+    margin: 0 0 10px 0;
+  }
+
+  main section.fundings div.fund-blocks {
+    display: flex;
+  }
+
+  section.content {
+    display: flex;
+    flex-wrap: wrap;
+    width: 98%;
+  }
+
+  section.last-deals ul li.flex > a.news-img {
+    display: block;
+    padding-top: 75%;
+  }
+
+  footer {
+    top: 200px;
+  }
+
+  footer div.legal {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  footer div.legal span {
+    display: inline-block;
+    margin: 5px;
+  }
+}
+
+/* Breakpoint w:1024px */
```

Comment:
Please remove **all** comments from your CSS. This will make your code look cleaner ✨